pkg/api/ops/ops.go

```diff
@@ -21,10 +21,28 @@ const (
 
 	MSG_OP_DATA = 24
 
+	MSG_OP_CGROUP = 25
+
 	// just for testing
 	MSG_OP_TEST = 254
 )
 
+const (
+	MSG_OP_CGROUP_UNDEF       = iota
+	MSG_OP_CGROUP_MKDIR       = 1
+	MSG_OP_CGROUP_RMDIR       = 2
+	MSG_OP_CGROUP_RELEASE     = 3
+	MSG_OP_CGROUP_ATTACH_TASK = 10
+)
+
+const (
+	CGROUP_UNTRACKED    = iota // Cgroup was created but we did not track it
+	CGROUP_NEW          = 1    // Cgroup was just created
```

Comment:
Might be worth adding a note that this is state is only used for debugging messages and information printed.

Reply:
Done.